docs/cufinufft_migration.rst

```diff
@@ -0,0 +1,99 @@
+.. _cufinufft_migration:
+
+Migration to cuFINUFFT v2.2
```

Comment:
We may want to think about the "cuFINUFFT" name, there for historical reasons, and whether to retire since it is now just the GPU part of FINUFFT. Fine for now. Would require more discussion.

Reply:
Opened issue #409. Can discuss there.